src/index.pug

```diff
@@ -11,6 +11,12 @@ html(lang='en')
       link(rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro")
     for script in scripts
       link(rel="preload", href=script, as="script")
+    script(src="https://cdnjs.cloudflare.com/ajax/libs/cookie-banner/1.2.2/cookiebanner.min.js",
```

Comment:
tried to install this in node_modules and include in CLI scripts[] but didnt take